packages/diagrams/lib/src/tabs.dart

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'diagram_step.dart';
+
+const Duration _kTabScrollDuration = Duration(milliseconds: 300);
```

Comment:
Uber nit here & below in naming - there is no scrolling in this animation

Reply:
The TabView animation is actually a scroll animation under the hood (I got too used to it from working on it recently). However, I do agree that it can be vague without knowing this. Would `tabChangeDuration` or `tabSwitchDuration` be better alternatives?

Reply:
Oh fun fact! It's not a huge deal. You can leave it if you prefer.